tracing.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor, ConsoleSpanExporter
+
+
+def get_provider():
+    provider = TracerProvider()
+    trace.set_tracer_provider(provider)
+    return provider
+
+
+def add_exporter(provider, exporter, processor=BatchSpanProcessor):
+    """Utility method to add an exporter.
+
+    We use the BatchSpanProcessor by default, which is the default for
+    production. This is asynchronous, and queues and retries sending telemetry.
+
+    In testing, we instead use SimpleSpanProcessor, which is synchronous and
+    easy to inspect the output of within a test.
+    """
+    # Note: BatchSpanProcessor is configured via env vars:
+    # https://opentelemetry-python.readthedocs.io/en/latest/sdk/trace.export.html#opentelemetry.sdk.trace.export.BatchSpanProcessor
+    provider.add_span_processor(processor(exporter))
+
+
+def setup_default_tracing():
+    provider = get_provider()
+
+    """Inspect environment variables and set up exporters accordingly."""
+    if "OTEL_EXPORTER_OTLP_HEADERS" in os.environ:
+        if "OTEL_SERVICE_NAME" not in os.environ:
+            raise Exception(
```

Comment:
I just changed job-runners behaviour here to default to "jobrunner" - I think we should do the same here, default to "airlock" if not set, rather than raise.

That makes it easier to share configuration in backend-server, as their default OTEL_SERVICE_NAME is correct for production.

Reply:
Yep, I'm just changing it to match your jobrunner changes